vmware/vmware-cloud-director-ui-components
# Adds tests to number with unit + deletes unused code

Signed-off-by: Ryan Bradford <rbradford@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:


## PR Type

What kind of change does this PR introduce?



-   [x] Other... Please describe: Testing!

## What does this change do?

1. Adds tests for number with unit form input. Also, there was a block of code in the `get displayName` that seemed to be un-callable and unused. I deleted this block.
2. VSCode (and potentially other editors) use the closest tsconfig to compile files. This means in the project `find-cypress-widget.ts` (which includes Cypress) was being added to the compilation of the project. This was causing errors to appear in the spec files. To get around this, I created a new tsconfig in just the /components directory that ecludes the find-cypress-widget.ts.

## What manual testing did you do?

1. Ran the examples app.
2. Navigated to the number with unit tab.
3. Changed the value to 4096 mb
4. Observed the display changing to 4096 mb
5. Set to readonly
6. Observed the correct display of 4096 mb
7. Change unit to GB
8. Set to 1 GB
9. Observe correct display of 1024 MB
10. Set to readonly
11. Observe correct display of 1024 MB


## Does this PR introduce a breaking change?
